Source/WebCore/page/InteractionRegion.cpp

```diff
@@ -564,9 +565,8 @@ std::optional<InteractionRegion> interactionRegionForRenderedRegion(RenderObject
             if (borderRadii.bottomRight().minDimension() == maxRadius)
                 maskedCorners.add(InteractionRegion::CornerMask::MaxXMaxYCorner);
         } else {
-            Path path;
-            path.addRoundedRect(roundedRect);
-            clipPath = path;
+            clipPath = borderShape.pathForOuterShape(renderBox->document().deviceScaleFactor());
+            WTF_ALWAYS_LOG("interactionRegionForRenderedRegion - rounded rect" << borderShape.deprecatedRoundedRect() << " device scale " << renderBox->document().deviceScaleFactor() << " " << clipPath);
```

Comment:
This should log to a channel

Reply:
I'm trying to debug the VisionOS failure.